mock-core-configs/etc/mock/centos-stream+epel-10-s390x.cfg

```diff
@@ -0,0 +1,8 @@
+config_opts["koji_primary_repo"] = "epel"
+include('templates/centos-stream-10.tpl')
+include('templates/epel-10.tpl')
+
+config_opts['root'] = 'centos-stream+epel-10-s3100x'
+config_opts['description'] = 'CentOS Stream 10 + EPEL'
+config_opts['target_arch'] = 's3100x'
+config_opts['legal_host_arches'] = ('s3100x',)
```

Comment:
s/s3100x/s390x/

Reply:
Fixed in 911ea87e250d4db37bd3ac009db0f1e043eece21